app/src/main/java/com/example/mydigitalcloset/Login.java

```diff
@@ -52,36 +65,32 @@ public void onClick(View view) {
                 username = String.valueOf(textInputEditTextUsername.getText());
                 password = String.valueOf(textInputEditTextPassword.getText());
 
-                //make sure every field is entered
-                if(!TextUtils.isEmpty(username) && !TextUtils.isEmpty(password)) {
-
+                if (!TextUtils.isEmpty(username) && !TextUtils.isEmpty(password)) {
```

Comment:
Good check- i've been meaning to implement a check like this for uploading clothing, so i'll make sure to utilize it.